crates/harp/src/exec.rs

```diff
@@ -348,19 +348,11 @@ where
 
     match res {
         Some(res) => res,
-        None => {
-            let mut err_buf = r_peek_error_buffer();
-
-            if err_buf.len() > 0 {
-                err_buf = format!("\nLikely caused by:\n{err_buf}");
-            }
-
-            Err(Error::TopLevelExecError {
-                message: format!("Unexpected longjump{err_buf}"),
-                backtrace: std::backtrace::Backtrace::force_capture(),
-                span_trace: tracing_error::SpanTrace::capture(),
-            })
-        },
+        None => Err(Error::TopLevelExecError {
+            message: String::from("Unexpected longjump"),
+            backtrace: std::backtrace::Backtrace::force_capture(),
+            span_trace: tracing_error::SpanTrace::capture(),
+        }),
     }
```

Comment:
There is a doc comment above that needs updating

```
/// If a longjump does occur for any reason (including but not limited to R
/// errors), the caller is notified, in this case by an `Err` return value
/// of kind `TopLevelExecError`. The error message contains the contents of
/// the C-level error buffer. It might or might not be related to the cause
/// of the longjump. The error also carries a Rust backtrace.
```

I'm not sure if it is worth mentioning something about stack overflows in that doc comment. Don't care either way.